packages/core/test/scxml.test.ts

```diff
@@ -156,19 +147,19 @@ const testGroups = {
     'test174.txml',
     'test175.txml',
     'test176.txml',
-    // 'test179.txml', // conversion of <content> in <sens> not implemented yet
+    // 'test179.txml', // conversion of <content> in <send> not implemented yet
     // 'test183.txml', idlocation not implemented yet
     'test185.txml',
-    // 'test186.txml', // not sure yet why
+    'test186.txml',
     'test187.txml',
     'test189.txml',
-    // 'test190.txml', // _sessionid not yet available for expressions
+    'test190.txml', // note: _sessionid is undefined for expressions
     'test191.txml',
-    // 'test192.txml', // conversion of #_invokeid not implemented yet
+    // 'test192.txml', // done.invoke inexact event descriptor
```

Comment:
isnt the removed comment still relevant? i don't think we convert sends' targets yet

Reply:
The more pertinent issue is that `done.invoke` is an inexact event descriptor.